interfaces/builtin/tpm.go

```diff
@@ -0,0 +1,67 @@
+package builtin
+
+import "github.com/snapcore/snapd/interfaces"
+
+var tpmPermanentSlotAppArmor = []byte(`
+# Description: for those who need to talk to the system TPM chip over /dev/tpm0
+# Usage: common
+
+/dev/tpm0 rw,
```

Comment:
Can we ever only have /dev/tpm0 or can we also have multiple? Maybe not really common so lets start with tpm0.